src/Middleware/HttpLogging/samples/Logging.W3C.Sample/Startup.cs

```diff
@@ -15,6 +15,10 @@ public void ConfigureServices(IServiceCollection services)
         {
             // Log all W3C fields
             logging.LoggingFields = W3CLoggingFields.All;
+            logging.AdditionalRequestHeaders = new[] {
```

Comment:
Could you share what the log file looks like after this change to the sample?

Reply:
```
#Version: 1.0
#Start-Date: 2022-05-20 18:31:34
#Fields: date time c-ip cs-username s-computername s-ip s-port cs-method cs-uri-stem cs-uri-query sc-status time-taken cs-version cs-host cs(User-Agent) cs(Cookie) cs(Referer) cs(x-forwarded-for) cs(x-client-ssl-protocol)
2022-05-20 18:31:34 127.0.0.1 - VVKTV23H53 127.0.0.1 5000 GET / - 200 19.767 HTTP/1.1 localhost:5000 curl/7.79.1 - - 1.1.1.1 TLS+1.2
```